### PR TITLE
Update goja

### PR DESCRIPTION
**Pull request by mstoykov**

### CODE REVIEW

**Comment by mstoykov** [comment]
See https://github.com/dop251/goja/issues/410 for found issues.

IMO none of them are particularly problematic. The eval ones might be nice to be fixed - but likely not used by anyone either way or so I hope :crossed_fingers:

File: core/engine_test.go [hunk 1]
```diff
@@ -967,7 +967,7 @@ func TestVuInitException(t *testing.T) {
 
 	var exception errext.Exception
 	require.ErrorAs(t, err, &exception)
-	assert.Equal(t, "Error: oops in 2\n\tat file:///script.js:10:9(30)\n\tat native\n", err.Error())
+	assert.Equal(t, "Error: oops in 2\n\tat file:///script.js:10:9(29)\n\tat native\n", err.Error())
```
Comment (olegbespalov) [inline core/engine_test.go:970]: Could it be `assert.ErrorContains` or something similar? :thinking:

Comment (mstoykov) [reply core/engine_test.go:970]: This is from 1.7.1 which I think we just update to. And I would prefer not to do more than the needed changes in update like that. 

But yeah - we can start replacing those calls. But it probably will be nicer if it's done through the whole codebase in one commit :shrug:

**Review by olegbespalov — APPROVED** [review]
:crossed_fingers:

File: core/engine_test.go (hunk 1 above)
Comment (mstoykov) [reply core/engine_test.go:970]: Actually is seems like it should be [`EqualError`](https://pkg.go.dev/github.com/stretchr/testify/require#EqualError) in this case

Comment (olegbespalov) [reply core/engine_test.go:970]: Whatever works best, I mean something that we shouldn't adjust with the updates :shrug:

Comment (codebien) [reply core/engine_test.go:970]: Maybe the others but this specific code is testing the exception so I think it's better to validate also the returned number.

**Review by na-- — DISMISSED** [review]
:heart: :tada:

**Review by na-- — APPROVED** [review]

**Review by codebien — APPROVED** [review]

**Pull request merged**